core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20210915_create_entity_cache_table.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="davidan" id="create_WORKSPACE_ENTITY_CACHE">
+        <createTable tableName="WORKSPACE_ENTITY_CACHE">
+            <column name="workspace_id" type="BINARY(16)">
+                <constraints primaryKey="true"
+                             nullable="false"
+                             referencedTableName="WORKSPACE" referencedColumnNames="id"
+                             foreignKeyName="FK_WEC_WORKSPACE" deleteCascade="true"/>
+            </column>
+            <column name="entity_cache_last_updated" type="TIMESTAMP(6)" defaultValueDate="1970-01-01 00:00:02">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
in practice, the code should never rely on the default value here. Still, I've chosen a default value that is one second later than the magic "cache is in error state" value. This should make it such that any workspace that somehow uses the db default value will be considered out of date and get its cache populated in an upcoming sweep.